src/service/action-impl.js

```diff
@@ -200,6 +203,10 @@ export class ActionService {
         this.addChangeDetails_(event);
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'input-debounced') {
+      this.root_.addEventListener('input', debounce_(event => {
+        this.trigger(dev().assertElement(event.target), name, event);
```

Comment:
The ones just a few lines above this one `if (name == 'keydown') { ... }`. Does anything make this case different than those?